pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -100,15 +100,58 @@ func checkHyperVServiceRunning() error {
 }
 
 func checkUserPartOfCrcUsersAndHypervAdminsGroup() error {
-	_, _, err := powershell.Execute(fmt.Sprintf("Get-LocalGroupMember -Group 'crc-users' -Member '%s'", username()))
+	groupMembers, _, err := powershell.Execute(`(Get-LocalGroupMember -Group 'crc-users').Name`)
 	if err != nil {
 		return err
 	}
-
+	logging.Debug("Checking current user part of 'crc-user' group")
```

Comment:
`Checking if the current user is in the 'crc-user' group` or `is part of the 'crc-user' group`